aries-site/src/themes/aries.js

```diff
@@ -6,20 +6,23 @@ export const aries = deepMerge(hpe, {
   global: {
     colors: {
       focus: 'teal!',
-      placeholder: 'text-xweak',
     },
     input: {
       weight: 500,
     },
   },
-  button: {
-    color: props =>
-      props.primary ? '#FFFFFF' : normalizeColor('text', props.theme),
-  },
   layer: {
     background: 'background',
   },
   checkBox: {
+    color: 'selected-text',
+    gap: 'small',
+    check: {
```

Comment:
There is a theme attribute of `checkBox.check.radius` that we can use here instead of extend.

Reply:
I missed that thanks